diagnostic.md

```diff
@@ -7,40 +7,42 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone https://github.com/jacobhereford/git-basics-diagnostic.git -->
```

Comment:
But on this one you specified URL not SSH is the reason why I used the HTTPS

Sent from my iPhone

On May 17, 2016, at 9:44 PM, Brian Berzellini <notifications@github.com> wrote:

In diagnostic.md:

> @@ -7,40 +7,42 @@ What command do we use to do that? Use the **correct** URL from your fork of
>  this repository in your answer.
>  
>  ```sh
> -<!-- Remove this comment and place your answer here. -->
> +<!-- git clone https://github.com/jacobhereford/git-basics-diagnostic.git -->
git clone git@github.com/jacobhereford/git-basics-diagnostic.git. You have the https url, we use the SSH.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub